var/spack/repos/builtin/packages/hsa-rocr-dev/package.py

```diff
@@ -34,6 +34,7 @@ class HsaRocrDev(CMakePackage):
     variant('image', default=True, description='build with or without image support')
 
     depends_on('cmake@3:', type="build")
+    depends_on('xxd', when='@4.2:', type='build')
```

Comment:
we have enabled image_support=true from 3.7.0 and i think xxd will need to be supported from 3.7.0 onwards

Reply:
Thanks, fixed that, can you re-review?